blog.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+
+  <body>
+
+    <header style="background-color:#ADD8E6;">
```

Comment:
You'll want to get rid of any inline styling.  For the next few weeks, make sure to migrate any styling to a css file.